content/02.main-text.md

```diff
@@ -2,12 +2,12 @@
 
 The internet enables science to be shared in real-time at a low cost to a global audience.
 This development has decreased the barriers to making science open, while supporting new massively collaborative models of research.
-However, the scientific community requires tools whose workflows encourage openness rather than stifle it.
-For example, manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, where anyone is able to contribute and the contribution history is preserved and public.
-Hence, we report a new tool and workflow for authoring scholarly manuscripts in the open, as well as the collaborative project that led to its creation.
+However, the scientific community requires tools whose workflows encourage openness.
```

Comment:
I saw a @dhimmel question on this in the PR into @agitter's repo. It is unclear that it's fair to describe current tools as stifling research. It seems more likely that they were simply designed without that use case in mind.

Reply:
@dhimmel wrote https://github.com/agitter/meta-review/pull/1#discussion_r206365062
> Is the point still clear without "rather than stifle it"?

I think the point is still sufficiently clear with the new phrasing.